test/core/generate_parse_fixture_yml.py

```diff
@@ -7,7 +7,7 @@
 from sqlfluff.core import FluffConfig
 from sqlfluff.cli.commands import quoted_presenter
 
-from dialects.parse_fixtures import get_parse_fixtures, load_file
+from test.dialects.parse_fixtures import get_parse_fixtures, load_file
```

Comment:
This import was already broken somehow. Could be a separate PR if this fix is needed sooner.